docs/user/hbase/usage.rst

```diff
@@ -32,7 +32,8 @@ The data store takes several parameters (required parameters are marked with ``*
 ===========================================  ======= ========================================================================================
 Parameter                                    Type    Description
 ===========================================  ======= ========================================================================================
-``hbase.catalog *``                          String  The name of the GeoMesa catalog table (previously ``bigtable.table.name``)
+``hbase.catalog *``                          String  The name of the GeoMesa catalog table, including the HBase namespace (if any) separated
```

Comment:
Are HBase namespaces available in both 1.4.x and 2.x?

Reply:
yeah, and our CI build runs the hbase tests against both hbase 1 and 2 so the api calls should work in both.